site/content/zh-hans/docs/developer-guide/plugin_development.md

```diff
@@ -15,7 +15,7 @@ HTNN 的插件分成两种：Go 插件和 Native 插件。Native 插件在运行
 5. 在 `./plugins/plugins/` 下创建一个目录，文件名必须和步骤一里面的名字一致。完成插件编写。不要忘记编写测试。如果您的插件简单，您可以仅编写集成测试。您可以以 `./plugins/plugins/demo` 为例。插件开发中涉及的 API 的文档位于这些 API 的注释当中。
 6. 在 `site/content/$your_language/docs/reference/plugins/$your_plugin.md` 中添加插件文档。您可以选择使用简体中文或英文编写文档，这取决于哪种是您擅长的语言。我们有 [工具](https://github.com/mosn/htnn/tree/main/site#cmdtranslator) 可以将其翻译成其他语言。
 7. 在 `./plugins/plugins.go` 中引入你的插件。前往目录 `./plugins`，运行 `make build-so`。插件将被编译入当前目录下的 `libgolang.so`。
-8. 在 `./plugins/tests/integration/` 中添加集成测试。有关如何运行集成测试，请阅读[插件集成测试框架](../plugin_integration_test_framework)。
+8. 在 `./plugins/tests/integration/` 中添加集成测试。有关如何运行集成测试，请阅读[插件集成测试框架](./plugin_integration_test_framework)。
```

Comment:
This is a bit tricky. The doc generated by the site builder is different from what it looks like on GitHub. You could take a look at the output result by building the website: https://github.com/mosn/htnn/blob/de1d13b57a48881edf83c1a03d62a2d38a1b821f/.github/workflows/website.yml#L39.

Reply:
Interesting. Both the version in this PR and the version in the main branch can pass the link check. (We have a link checker in the CI, you can learn more from https://github.com/mosn/htnn/pull/600)

Look like an absolute link is generated:
```
<li>在 <code>./plugins/tests/integration/</code> 中添加集成测试。有关如何运行集成测试，请阅读<a href="/zh-hans/docs/developer-guide/plugin_integration_test_framework/">插件集成测试框架</a>。</li>
```

Could you put this change in another PR?

Reply:
> Interesting. Both the version in this PR and the version in the main branch can pass the link check. (We have a link checker in the CI, you can learn more from #600)
> 
> Look like an absolute link is generated:
> 
> ```
> <li>在 <code>./plugins/tests/integration/</code> 中添加集成测试。有关如何运行集成测试，请阅读<a href="/zh-hans/docs/developer-guide/plugin_integration_test_framework/">插件集成测试框架</a>。</li>
> ```
> 
> Could you put this change in another PR?

Thank you, I will go and check out this PR.